01week/rockPaperScissors.js

```diff
@@ -9,11 +9,37 @@ const rl = readline.createInterface({
 
 
 function rockPaperScissors(hand1, hand2) {
+  hand1=hand1.toLowerCase().trim();
+  hand2=hand2.toLowerCase().trim();
 
-  // Write code here
-
+  if (hand1 === hand2){
+    return "It's a tie!"
+  }
+  else if (hand1==="rock" && hand2==="scissors") {
+    return "Hand one wins!"
+  }
+  else if (hand1==="paper" && hand2==="rock") {
```

Comment:
Could you also do else if ( condition || condition){ return "statement"} To make this look clean or more readable you could write it like this: 
else if ( (hand1 === "rock" && hand2 === "scissors) ||
(hand1 === "scissors" && hand2 === "paper")
.... enter next condition){
return "Hand one Wins! "